app/views/users/phone_confirmation/show.html.slim

```diff
@@ -2,17 +2,16 @@
 
 h1.heading = t('forms.phone_confirmation.header_text')
 p
-  | A confirmation code has been sent to <strong>#{@unconfirmed_mobile}</strong>.
-    Please enter it below:
+  'A confirmation code has been sent to <strong>#{@unconfirmed_mobile}</strong>.
```

Comment:
I actually think the old version is better and we should be using | for paragraphs across the board.

the `'` method is special in that it always ends the string with a space.  If you look a the html you will see an extra space before the </p> I think.